demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -526,3 +539,19 @@ def update_api_modules_dependents_rn(_pack, pre_release, update_type, added, mod
                                   modified_files_in_pack={integration_path}, pre_release=pre_release,
                                   added_files=set(), pack=integration_pack)
         update_pack_rn.execute_update()
+
+
+def check_docker_image_changed(added_or_modified_yml):
+    try:
+        diff = run_command(f'git diff origin/master -- {added_or_modified_yml}', exit_on_error=False)
+    except RuntimeError as e:
+        if any(['is outside repository' in exp for exp in e.args]):
+            return False, ''
+        else:
+            raise
```

Comment:
dont raise the error - print something to the user about being unable to check the docker image and skipping